topics/climate/tutorials/ocean-variables/tutorial.md

```diff
@@ -243,21 +477,67 @@ Now, if you have finished with your analysis you can exit ODV. To do so you need
 
 > <tip-title>Check Datatype</tip-title>
 > Go to your output 'data_from_Eutrophication_Med_profiles_2022_unrestricted'
-> 
+>
 > Check if the datatype is **netcdf** if not :
-> 
+>
 > {% snippet  faqs/galaxy/datasets_change_datatype.md %}
 {: .tip}
 
-## Launch DIVAnd 
+{% include _includes/cyoa-choices.html option1="New-user" option2="Experimented-user" default="New-user" disambiguation="userlevel"
```

Comment:
ok, and apparently it also doesn't work well if there are spaces in the option names, so I put a dash instead